background.ts

```diff
@@ -571,6 +571,10 @@ Electron.ipcMain.on('factory-reset', async(event, keepSystemImages) => {
   await settings.clear();
 
   Electron.app.quit();
+}
+
+Electron.ipcMain.on('factory-reset', async(event, keepSystemImages) => {
+  await doFactoryReset(keepSystemImages);
```

Comment:
Given that we don't do anything else, is there a point in `await`ing the result?

Reply:
True, but we should still catch any error and display it via console rather than let it disappear (because it will originate on an event handler).